mws/apis/reports.py

```diff
@@ -284,13 +270,10 @@ def update_report_acknowledgements(self, report_ids=None, acknowledged=None):
         """
         report_ids = report_ids or []
         if acknowledged is not None:
-            acknowledged = str(bool(acknowledged)).lower()
-        data = {
-            "Action": "UpdateReportAcknowledgements",
-            "Acknowledged": acknowledged,
-        }
-        data.update(utils.enumerate_param("ReportIdList.Id.", report_ids))
-        return self.make_request(data)
+            acknowledged = bool(acknowledged)
```

Comment:
Just to confirm: This has now changed to a bool, and no longer using a string?

Reply:
Yes. The cleaning utilities take care of converting bools to proper strings, so it's best to use the real data type here, rather than perform the extra work of writing a string manually.

This *may* create some havoc if someone enters `'false'`, which converts to `True` when run through `bool()`, so it might not make sense to actually do this conversion.

Reply:
Safer to add a helper function thats called at this point? (`aws_bool()`)  to deal with that kind of logic?

Reply:
Fair point, I've seen that pattern in some other contexts.

Added utility for `coerce_to_bool`, which coverts to a boolean and accepts certain string values as "false" for consistency.

Made slight adjustments in the API methods to accommodate. Couple spots had `'false'` as arg defaults, so I changed those to `False`; and in all those spots they perform a simple `is None` test before then calling `coerce_to_bool`.